_components/validation/validation.md

```diff
@@ -0,0 +1,26 @@
+---
+category: Components
+component:
+  status: ready
+  package: usa-validation
+  dependencies:
+lead: Stating validation requirements up front, with live feedback, means users won't be left guessing.
+permalink: /components/validation/
+title: Validation
+type: component
+subnav:
+  - text: Preview
+    href: '#validation-preview'
+  - text: Code
+    href: '#validation-code'
+  - text: Guidance
+    href: '#validation-guidance'
+  - text: Package
+    href: '#validation-package'
+tags:
+  - forms
+  - error
+  - check
+  - checking
+  - checkmark
+---
```

Comment:
Seeing a visual regression in Validation

## New
![image](https://user-images.githubusercontent.com/3385219/109692236-e59e7f00-7b4d-11eb-9515-20a5b5333b0b.png)

## Current
![image](https://user-images.githubusercontent.com/3385219/109692353-06ff6b00-7b4e-11eb-8d5e-057509bcec45.png)

Reply:
FIXED